api/v1beta1/keystoneendpoint.go

```diff
@@ -190,3 +190,52 @@ func DeleteKeystoneEndpointWithName(
 
 	return nil
 }
+
+// GetKeystoneEndpointList func
+func GetKeystoneEndpointList(
+	ctx context.Context,
+	h *helper.Helper,
+	namespace string,
+) (*KeystoneEndpointList, error) {
+
+	ke := &KeystoneEndpointList{}
+	err := h.GetClient().List(ctx, ke)
+	if err != nil {
+		return nil, err
+	}
+
+	return ke, nil
+}
+
+// GetKeystoneEndpointUrls returns all URLs currently registered. Visibility can be admin, public or internal.
+// If it is nil, all URLs are returned
+func GetKeystoneEndpointUrls(
+	ctx context.Context,
+	h *helper.Helper,
+	namespace string,
+	visibility *string,
+) ([]string, error) {
+
+	ke := &KeystoneEndpointList{}
+	err := h.GetClient().List(ctx, ke)
```

Comment:
same question on namespace filtering, could just call GetKeystoneEndpointList()

Reply:
Yes indeed, we should be calling GetKeystoneEndpointList